src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.Session_FilterModel.cs

```diff
@@ -221,12 +221,20 @@ internal partial class Session
 
             private Boolean IsAfterDot(Model model, ITextSnapshot textSnapshot, Dictionary<TextSpan, string> textSpanToText)
             {
-                var span = model.OriginalList.Span;
```

Comment:
Any chance we'd crash if you bring up completion in an empty file (or empty debugger intellisense window)?  Can you just try out those cases?